weblab/templates/registration/myaccount.html

```diff
@@ -0,0 +1,76 @@
+{% extends "base.html" %}
+{% load oauth %}
+
+{% block content %}
+  <h1>Your account</h1>
+
+  <form method="POST" action="">
+    {% csrf_token %}
+
+    <table id="myaccounttable" class="leftright">
+      <tr>
+        <th>Email address</th>
+        <td>
+          {% if form.email %}
+            {{ form.email }}
+            <span class="errorlist">{{ form.email.errors }}</span>
+          {% else %}
+            {{ user.email }}
+            <small>
+              Your account is connected to
+              {% for backend in backends.associated %}
+                {% if forloop.first %}{% elif forloop.last %}and{% else %}, {% endif %}
+                {{ backend|backend_name }}
+              {% endfor %}
+              so email address cannot be edited
+            </small>
+          {% endif %}
+        </td>
+      </tr>
+      <tr>
+        <th>Full name</th>
+        <td>{{ user.full_name }}</td>
+      </tr>
+      <tr>
+        <th>Institution</th>
+        <td>
+            {{ form.institution }}
+            <span class="errorlist">{{ form.institution.errors }}</span> 
+        </td>
+      </tr>
+      <tr>
+        <th>Permissions</th>
+        <td>
+          {% if user.is_super_user %}
+            Administrator - you have full administrative privileges
+          {% else %}
+            None
+          {% endif %}
+        </td>
+      </tr>
+      <tr>
+        <th>Registered since</th>
+        <td><time>{{ user.date_joined|date:"Y-m-d H:i:s" }}</time></td>
+      </tr>
+      <tr>
+        <th>Email preferences</th>
+        <td>
+          {{ form.receive_emails }}
+          {{ form.receive_emails.label }}
+          <span class="errorlist">{{ form.receive_emails.errors }}</span>
+        </td>
+      </tr>
+    </table>
+
+    <p>
+      <button>Update account</button>
+      {% if user.has_usable_password %}
+        <a class="button" href="{% url "password_change" %}">Change Password...</a>
```

Comment:
Interestingly, this has subtly different display from a standard button, at least on Firefox - it's a smidgen taller!